docs/usage/exceptions.rst

```diff
@@ -75,6 +75,14 @@ Litestar handles all errors by default by transforming them into **JSON response
 :class:`~litestar.exceptions.http_exceptions.HTTPException`, the responses will include the appropriate ``status_code``.
 Otherwise, the responses will default to ``500 - "Internal Server Error"``.
 
+
+The following handler for instance will default to ``MediaType.TEXT`` so the exception will be raised as text.
+
+.. literalinclude:: ../../tests/e2e/test_exception_handlers/test_exception_handlers.py
```

Comment:
Can we add this to `examples`? Otherwise we'll end up breaking the docs just by changing some tests 👀